code/game/objects/items/weapons/implants/implant_stealth.dm

```diff
@@ -0,0 +1,161 @@
+/**
+ * # Stealth Implant
+ *
+ * Implant which allows you to summon an MGS-style cardboard box that turns you invisble after a short delay.
+ */
+/obj/item/implant/stealth
+	name = "S3 bio-chip"
+	desc = "Allows you to be hidden in plain sight."
```

Comment:
```suggestion
	desc = "Allows you to be hidden in plain sight."
	implant_state = "implant-syndicate"
```
should probably have the syndicate coloration for this one. 
if you ever get a chance, you can also implement some cool flavor lore for it
```dm
implant_data = /datum/implant_fluff/stealth
```

Reply:
Done. Also added some some implant fluff.